### PR TITLE
Craftable Switchblades: The Finale

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
**AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA**
Adds Crafted Switchblades, an easily concealed retractable self defense tool.

## Why It's Good For The Game
More variety when it comes to petty crime, weapons crafting, self defense tools, and antaging! Helps aid realism as not many people would want to carry around an entire _Butcher's Cleaver_, but may be more inclined to go through the effort of getting a Switchblade.

## Changelog
:cl:
add: Craftable Switchblades, a weaker subtype of normal switchblades that can be crafted using a Kitchen Knife, Modular Receiver, Rifle Stock and some Cable Coil. Requires a welder to complete.
/:cl:
_God please let this work this time, I'm so very tired._

### CODE REVIEW

**Comment by ShadeAware** [comment]
God please let this work this time without issue.

**Comment by ShadeAware** [comment]
Also I fixed the throwing code bit so everything should work fine now. Keyword is _should_
 but fucking nothing else has been so far so I don't fucking know haha :joy:

**Comment by Arturlang** [comment]
You are pring from your master which isnt reccomended, also, please stop with making excessive new prs and actually use git

**Comment by ShadeAware** [comment]
I'm trying yet failing

**Comment by ghost** [comment]
15 damage pocket knife, yikes
looks fun

**Comment by ShadeAware** [comment]
### **ARE YOU PROUD OF ME NOW GHOMMIE?**

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -337,6 +339,14 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	user.visible_message("<span class='suicide'>[user] is slitting [user.p_their()] own throat with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
+/obj/item/switchblade/makeshift //It took me so fucking long to make this not shitcode holy FUCK. GODDAMN IT. SHIT. FUCK. NI-
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm]: No need to be so upset-ti in the comment, my gucci.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Sprites are roughly the same as the base switchblade, nothing makeshift-looking.

**Comment by ShadeAware** [comment]
> Sprites are roughly the same as the base switchblade, nothing makeshift-looking.

The switchblades are supposed to keep the same style, so theres more reason to use them besides being concealed.
Tbh over time these have become less makeshift and more craftable so I _might_ do a name change to clear up the confusion.

**Comment by ShadeAware** [comment]
Did a name change from makeshift to crafted because I refuse to compromise on the sprite. Its too sexy to not include.

**Comment by ShadeAware** [comment]
### The makeshift sprites cometh; down with Security, the Grey Tide rises in Wood and Steel.

**Comment by ShadeAware** [comment]
Travis does not like me very much does he.

**Comment by Ghommie** [comment]
No, this time that was my mistake with unbalanced parenthesis.
Please be careful when web-editing.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**